de.tu_bs.cs.isf.e4cf.parts.project_explorer/src/de/tu_bs/cs/isf/e4cf/parts/project_explorer/controller/ProjectExplorerToolBarController.java

```diff
@@ -91,26 +91,23 @@ public ProjectExplorerToolBarController(ToolBar toolbar, IEclipseContext context
 		btnNewFolder = createToolbarButton("Create New Folder", FileTable.NEWFOLDER_PNG, actionEvent -> {
 			NewFolderHandler handler = new NewFolderHandler();
 			if (handler.canExecute(services.rcpSelectionService)) {
-				handler.execute(shell, services.dialogService, services.rcpSelectionService, services.imageService,
-						services.workspaceFileSystem);
+				handler.execute(services.rcpSelectionService);
 			}
 		});
 
 		// File Actions
 		btnNewFile = createToolbarButton("Create new File", FileTable.NEWFILE_PNG, actionEvent -> {
 			NewFileHandler handler = new NewFileHandler();
 			if (handler.canExecute(services.rcpSelectionService)) {
-				handler.execute(context, services.dialogService, services.rcpSelectionService, services.imageService,
-						services.workspaceFileSystem);
+				handler.execute(services.rcpSelectionService, services.imageService, services.workspaceFileSystem);
 			}
 		});
 
 		// Import Files
 		btnImportFiles = createToolbarButton("Import ...", FileTable.FILE_PNG, actionEvent -> {
```

Comment:
The displayed text should not imply the possibility of importing directories if the user can not do it. Change to "Import Files" here and in context menu

Reply:
updated